### PR TITLE
Major, and I'm talking MAJOR... ... ... cleanup

### PR DESCRIPTION
**Pull request by himedlooff**
Like major, like don't freak out.

:see_no_evil::hear_no_evil::speak_no_evil:

I pushed to my gh-pages branch:
- http://himedlooff.github.io/cf-grid/docs/
- http://himedlooff.github.io/cf-grid/custom-demo/

Just some of the updates, can't remember them all:
- Ported to use cf-grunt-config.
- Generates docs.
- Replaced the name Ghost with cf-grid everywhere, it's confusing.
- demo has been removed and instead renamed examples to custom-demo. I did not use demo because I did not want it to get confused with the topdoc generated demo.
- Some refactoring, notably I removed the .col mixin and simply placed it inside of column because it seems to overly complicate things. Let me know if there was a good reason for this. I also simplified the demo a little and removed responsive stuff from it since they were not ideal methods for using this responsively.

### CODE REVIEW

File: src/cf-grid.less [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+      - "If you want IE 6/7 support, ensure boxsizing.htc is in the same dir as your stylesheet.
+        Read more: https://github.com/Schepp/box-sizing-polyfill."
+    family: "cf-grid"
+    patterns:
+    - name: "Less mixin"
+      codenotes:
+        - ".wrapper( @wrapper-width: @wrapper-width )"
+      notes:
+        - "You can create multiple wrappers of different max-widths by passing in
+          a value as the first argument."
```
Comment (Scotchester) [inline src/cf-grid.less]: This comment is a little confusing. How about just "You can create wrappers with different max-widths by passing a pixel value into the mixin."

Comment (Scotchester) [inline src/cf-grid.less]: I guess it's a little less confusing when I look at the usage examples below.

Comment (himedlooff) [inline src/cf-grid.less]: I don't mind changing it.

File: src/cf-grid.less [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+      - "Computes column widths and prefix/suffix padding."
+      - "CSS borders are used for fixed gutters."
+    - name: "Usage"
+      codenotes:
+      - |
+        .main-wrapper {
+          .wrapper();
+        }
+        .half {
+          .column(1, 2);
```
Comment (Scotchester) [inline src/cf-grid.less:110]: Wow! This kind of usage never occurred to me.

Comment (himedlooff) [inline src/cf-grid.less:110]: Do you mean that you can create layout configurations on the fly?

Comment (Scotchester) [inline src/cf-grid.less:110]: Yeah.

File: src/cf-grid.less (hunk 1 above)
Comment (himedlooff) [inline src/cf-grid.less]: Changed.

File: src/cf-grid.less (hunk 2 above)
Comment (himedlooff) [inline src/cf-grid.less:110]: Yeah it's not just a single grid pony. I'm assuming this would be a major dependency for the forth coming layout component.

Comment (Scotchester) [inline src/cf-grid.less:110]: Great stuff!

**Pull request merged**